project/about.md

```diff
@@ -0,0 +1,100 @@
+# About Pacifica
+
+## History
+
+Pacifica was created from a collaboration in the
+[United States National Laboratory](https://en.wikipedia.org/wiki/United_States_Department_of_Energy_national_laboratories)
+system. The Environmental Molecular Sciences Laboratory ([EMSL](https://www.emsl.pnnl.gov))
+collaborated with its host institution Pacific Northwest National Laboratory
+([PNNL](https://www.pnnl.gov)) to help manage the scientific data generated
+by researchers.
+
+EMSL is a [user facility](https://www.energy.gov/science/science-innovation/office-science-user-facilities)
+sponsored by the Department of Energy ([DOE](https://www.energy.gov))
+Office of Science ([here](https://www.energy.gov/science)) Biological and
+Environmental Research ([BER](https://www.energy.gov/science/ber/biological-and-environmental-research)).
+
+Pacifica was created by EMSL and PNNL in response to 
+[DOE Data Management Policy](https://www.energy.gov/datamanagement/doe-policy-digital-research-data-management)
+to manage open research data to make it more transparent. EMSL and
+PNNL both have similar administrative problems in that a single
+project can not shoulder the burden of meeting the policy on its
+own. This brought both EMSL and PNNL to the table to collaborate on
+developing an institutional data management system for open research
+science.
+
+## The Data Management Challenge
+
+The challenge is expressed by the nexus of the operating model of a
+large institution and data policies its required to follow. 
+
+### DOE Data Management Policy
+
+The DOE Data Management policy covers a lot, but we will only highlight
+the guiding principles here.
+
+#### Effective Data Management
+
+```
+ 1. Effective data management has the potential to increase the pace of
+    scientific discovery and promote more efficient and effective use of
+    government funding and resources. Data management planning should be
+    an integral part of research planning.
+```
+
+The imparative to increase the pace of scientific discovery in 
+conjunction with being more efficient with the resources we use is
+increasingly becoming more important. Pacifica has solutions for
+these problems and would like to increase its impact in solving this
+challenge.
+
+#### Sharing Proves Integrity
+
```

Comment:
Any chance there is some punctuation missing in the statement below? It feels like it really needs some.

Reply:
https://www.energy.gov/datamanagement/doe-policy-digital-research-data-management#Principles bullet number 2 is quite long.